### PR TITLE
Local CMS backup

### PR DESCRIPTION
**Pull request by ddjnw1yu**
Temporary solution for the issue(https://github.com/ABI-Software/12-labours-app/issues/82)
Using local content if don't pass the csm environment variable.
Currently, it will still require for manually running the script to pull the latest cms content before deployment/build docker image.

### CODE REVIEW

File: plugins/graphcms.js [hunk 1]
```diff
@@ -9,5 +9,8 @@ export default (
       nuxtState.nuxtState.config.graphcms_endpoint = graphcms_endpoint;
     });
   }
-  inject("graphcms", new GraphQLClient(graphcms_endpoint));
+  inject(
+    "graphcms",
+    graphcms_endpoint ? new GraphQLClient(graphcms_endpoint) : graphcms_endpoint
```
Comment (alan-wu) [inline plugins/graphcms.js]: Clearer when explicitly assign the variabler as `graphcms_endpoint ? new GraphQLClient(graphcms_endpoint) : undefined`.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Looks good, please see comment for a single line change please.

**Comment by ddjnw1yu** [comment]
> Looks good, please see comment for a single line change please.

`Undefined` is not available here. I have updated the if statement in **_graphcmsQuery.js_**, that probably will be a better way.

**Review by alan-wu — APPROVED** [review]
Looks good, thanks.

**Pull request merged**